src/item/lookup.js

```diff
@@ -24,20 +24,20 @@ define([
 		// 1: Due to #16, never use the cached resolved non-leaf nodes. It may not
 		//    represent its leafs in its entirety.
 		value = resourceGet( Cldr._resolved, normalizedPath );
-		if ( value && typeof value !== "object" /* 1 */ ) {
+		if ( typeof value === "string" /* 1 */ ) {
```

Comment:
You are right, there could potentially be booleans or numbers in addition to strings.